npm-client/src/create_order_instruction.ts

```diff
@@ -91,8 +96,11 @@ export async function buildOrderInstruction(
   forOutcome: boolean,
   price: number,
   stake: BN,
-  priceLadderPk?: PublicKey,
-  productPk?: PublicKey,
+  options: {
+    priceLadderPk?: PublicKey;
+    productPk?: PublicKey;
+    expiresOn?: number;
```

Comment:
Perhaps like stake we should make this a BN and not do the conversion ourselves 🤔